### ISSUE TITLE
Not working with configuration file.

### ISSUE THREAD

**Issue by Ved-un** [Raniconduh/cscroll#5]
Configuration file:   

```
color = true
dots = true
icons = true
long = true
opener = xdg-open
```

Terminal output:    
```
[elem] ~ ❯ cscroll                                                                      (08:21:45)
zsh: segmentation fault (core dumped)  cscroll
[elem] ~ ❯
```

**Comment by Raniconduh** [comment]
This is not the issue but you are forgetting to quote `xdg-open` (you don't even need to specify this on systems other than mac)

**Comment by Ved-un** [comment]
I don't understand you - what am I forgetting?    
I changed my config and the same thing happens 
```
color = true
dots = true
icons = true
long = true
# opener = xdg-open
```

**Comment by Ved-un** [comment]
... and even without that     
```
color = true
dots = true
icons = true
long = true
```
error
`zsh: segmentation fault (core dumped)  cscroll`

**Comment by Ved-un** [comment]
Now I checked and it happens only when I use _dots = true_ in the config,    
and without this setting everything is fine.     

It works like this without any problems
```
color = true
# dots = true
icons = true
long = true
```

**Comment by Raniconduh** [comment]
This problem should have been fixed as of ebadd77

Are you sure you're running an updated version?

**Comment by Ved-un** [comment]
Now I've updated and everything is fine.    

Thank you.
